src/tests/test_resultset.py

```diff
@@ -271,6 +271,20 @@ def test_convert_to_dataframe_select(session, request, mock_config, statement):
     assert df.to_dict() == {"x": {0: 1, 1: 2, 2: 3, 3: 4, 4: 5}}
 
 
+@pytest.mark.parametrize(
+    "session, expected_value",
+    [
+        ("with a as (select * from b) select * from b", True),
```

Comment:
try with a CTE that has multiple with ... definitions

Reply:
returns select on this: sqlparse.parse('WITH CTE1 AS (SELECT * FROM table1), CTE2 AS (SELECT * FROM table2) SELECT * FROM CTE1 JOIN CTE2 ON CTE1.id = CTE2.id;')[0].get_type()